native-netty-plot/pom.xml

```diff
@@ -8,20 +8,19 @@
   <name>netty-plot</name>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <graalvm.version>21.0.0.2</graalvm.version>
   </properties>
 
   <dependencies>
     <dependency>
-      <groupId>org.graalvm.nativeimage</groupId>
-      <artifactId>native-image-maven-plugin</artifactId>
-      <version>${graalvm.version}</version>
+     <groupId>org.graalvm.buildtools</groupId>
```

Comment:
@LesiaChaban, this is not the way to specify the Maven plugin for Native Image building. See how to https://graalvm.github.io/native-build-tools/latest/maven-plugin-quickstart.html > step 6.